docs/articles/manageSynapseCredentials.html

```diff
@@ -114,7 +123,7 @@ <h2 class="hasAnchor">
 <h3 class="hasAnchor">
 <a href="#one-time-login" class="anchor"></a>One Time Login</h3>
 <p>Use username and password to login as follows:</p>
-<pre><code><a href="https://www.rdocumentation.org/packages/base/topics/library">library("synapser")
+<pre><code><a href="https://www.synapse.org">library("synapser")
```

Comment:
I'm not sure what caused this link (and other similar ones) to change, but I don't think that linking to Synapse here makes much sense.

Reply:
Hmm, this seem to be the result of this change in pkgdown:
https://pkgdown.r-lib.org/news/#auto-linking-improvements

_library(pkgdown) is now automatically linked to the reference index for “pkgdown” not the documentation for library()_

Reply:
Ah ok 👍